pkg/cluster/util.go

```diff
@@ -0,0 +1,225 @@
+package cluster
+
+import (
+	"archive/zip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/k3s/pkg/daemons/config"
+)
+
+const (
+	backupDest                 = "/tmp/rke1"
```

Comment:
Would it be safer to use `ioutil.TempDir` since it will get the temp dir from the operation system and make this safter/more portable?